.env.example

```
@@ -0,0 +1,6 @@
+# Copy this file to .env and fill in your values
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=changeme
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)